content/en/docs/tasks/observability/gateways/test_utils.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
How about moving this to `tests/util/addons.sh`? In the future we can refactor and make it more generally reusable for other tests that need addons.

Reply:
Done. I made the function more flexible. (I'm not a bash expert. So if you know a better implementation than using `case`..`esac` then please suggest).